NullableAttributes.cs

```diff
@@ -0,0 +1,155 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
Added as a loose file that is included by projects needing it. That's probably not the best solution on the long run but until we find the right way to have this shimlayer/polyfill project I want to start with something simple.